apps/api/Sources/App/Services/LegacyRest/AppEditions.swift

```diff
@@ -20,12 +19,12 @@ extension LegacyRest {
   }
 
   private static func queryData(lang: Lang) async throws -> Data {
-    let editions = try await Current.db.query(Friend.self).all()
+    let editions = try await Friend.Joined.all()
       .filter { $0.lang == lang }
-      .flatMap { try $0.documents.models }
+      .flatMap(\.documents)
```

Comment:
What's the `\` syntax?

Reply:
keypaths - one of my favorite swift features, but a bit hard to grok initially because basically no other language has them, see:

https://www.swiftbysundell.com/articles/the-power-of-key-paths-in-swift/
https://www.swiftbysundell.com/tips/passing-key-paths-as-functions/